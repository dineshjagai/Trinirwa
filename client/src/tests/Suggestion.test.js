import React from 'react';
import renderer from 'react-test-renderer';
import Suggestions from '../components/Suggestion';

test('Test Temp snapshot', () => {
  const component = renderer.create(<Suggestions />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
