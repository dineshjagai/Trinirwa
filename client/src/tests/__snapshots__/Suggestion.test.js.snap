// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Temp snapshot 1`] = `
<div
  className="box-container"
>
  <div
    className="title"
    style={
      Object {
        "color": "#0C8367",
      }
    }
  >
    Suggestions
  </div>
  <br />
  <div
    className="box"
  >
    <div
      className="containers"
    />
  </div>
</div>
`;
